README.md

```diff
@@ -91,9 +91,18 @@ The Makefile provides several targets:
   * *test-short*: run the short tests
   * *format*: format the source code
   * *vet*: check the source code for common errors
-  * *docker*: build a docker container for the current `HEAD`
   * *assets*: build the new experimental React UI
 
+### Building the Docker image
+
+The `make docker` target is designed for use in our CI systemd.
+You can build a docker image locally with the following commands:
+
+    $ promu crossbuild -p linux/amd64
```

Comment:
you still need to build the react assets

Reply:
When I ran `promu crossbuild -p linux/amd64` it did it for me:

```
➜  prometheus git:(master) promu crossbuild -p linux/amd64
> running the base builder docker image
Unable to find image 'quay.io/prometheus/golang-builder:1.15-base' locally
1.15-base: Pulling from prometheus/golang-builder
62dfdff96a79: Pull complete 
79deb48d05f4: Pull complete 
afe63264fea1: Pull complete 
0187e2a6ae46: Pull complete 
067d7b40510c: Pull complete 
46aeee8f0bd1: Pull complete 
Digest: sha256:e1306241a24129abf01ee2c81ed1003db8b3cd91c738228ac1661cf162ad1a09
Status: Downloaded newer image for quay.io/prometheus/golang-builder:1.15-base
f34d5332fa7b7214b713d0768d183b1e8f1bae6691c615601a605ea56d833c77
# linux-amd64
>> building React app
building React app
yarn run v1.22.5
$ react-scripts build
Creating an optimized production build...
Compiled successfully.

File sizes after gzip:

  285.88 KB  build/static/js/2.14ef932b.chunk.js
  30.64 KB   build/static/js/main.032f0151.chunk.js
  23.98 KB   build/static/css/2.300141ce.chunk.css
  1.5 KB     build/static/css/main.0b010d50.chunk.css
  769 B      build/static/js/runtime-main.7e00e584.js

The project was built assuming it is hosted at ./.
You can control this with the homepage field in your package.json.

The build folder is ready to be deployed.

Find out more about deployment here:

  bit.ly/CRA-deploy

Done in 39.74s.
```

Reply:
you probably want to tell how to get promu (make promu?)